docs/source/development/traffic_ops.rst

```diff
@@ -229,30 +229,11 @@ To install the Traffic Ops Developer environment:
 	.. seealso:: `PostgreSQL instructions on setting up a database <https://wiki.postgresql.org/wiki/First_steps>`_.
 
 
-#. Use the ``setup`` command of the ``admin.pl`` script (see `Database Management`_ for usage) to set up the traffic_ops database(s).
-
-	 Example Output::
-
-		$ db/admin.pl --env=development setup
-		Using database.conf: conf/development/database.conf
-		Using database.conf: conf/development/database.conf
-		Using database.conf: conf/development/database.conf
-		Using database.conf: conf/development/database.conf
-		Using database.conf: conf/development/database.conf
-		Using database.conf: conf/development/database.conf
-		Executing 'drop database to_development'
-		Executing 'create database to_development'
-		Creating database tables...
-		Warning: Using a password on the command line interface can be insecure.
-		Migrating database...
-		goose: migrating db environment 'development', current version: 0, target: 20150210100000
-		OK    20141222103718_extension.sql
-		OK    20150108100000_add_job_deliveryservice.sql
-		OK    20150205100000_cg_location.sql
-		OK    20150209100000_cran_to_asn.sql
-		OK    20150210100000_ds_keyinfo.sql
-		Seeding database...
-		Warning: Using a password on the command line interface can be insecure.
+#. Use the ``reset`` and ``upgrade`` commands of the ``admin`` binary (see `Database Management`_ for usage) to set up the traffic_ops database(s).
+
+	 Example Usage::
```

Comment:
The reason I left this as `Example Usage::` was because it included output and I didn't know all of the parsers available to me at the time. Your's is a shell command, so instead of just the generic `::` it'd be best if you could use `.. code-block:: shell` like so:
```rst
.. code-block:: shell
    :caption: Example Usage

    db/admin --env=development reset && db/admin --env=development upgrade
```

Reply:
Not sure, I'll check it out.